.cspell_dict.txt

```diff
@@ -11,14 +11,17 @@ Daversin
 docname
 docnames
 Dokken
+DOLFI
```

Comment:
Do we really need `DOLFI` as a word?

Reply:
Not sure how that happend. I agree, we should remove this.